imap_processing/tests/idex/test_idex_l2a.py

```diff
@@ -242,3 +244,40 @@ def test_analyze_peaks_perfect_fits():
         assert np.allclose(fit_params[idx], np.asarray([mu, sigma, lam]), rtol=1e-12)
         # Test that there is a value greater than zero at this index
         assert area_under_curve[idx] > 0
+
+
+def test_estimate_dust_mass_no_noise_removal():
+    """
+    Test that estimate_dust_mass() is fitting the signal properly when there is no
+    noise removal.
+    """
+    pass
```

Comment:
Turn this into a `pytest.xfail()` mark for now so it doesn't count as a "good" test.